### ISSUE TITLE
Add contact info to Archives data page

### ISSUE THREAD

**Issue by MomoMoses** [johnjung/bmrcportal#175]
On the Archives data page that currently has the following fields:

- Name (of institution)
- Address (street address, city, state, zip)
- Link to repository website for collections
- Logo (already uploaded to Wagtail images)
- Spotlight blurb
- Finding aid prefix
- Is Member checkbox
- Order,
add contact information for archivists responsible for creating finding aids and/or reference librarians or archivists with whom the BMRC will want to contact for obtaining updated and new finding aids as they are completed and processed for upload.

- Name of archivist/librarian
- Title
- Email address
- Phone number
- Notes: can include what they are responsible for in more detail; should be something that changes relatively infrequently

Create a primary and secondary data column. Secondary column may be left blank.

EXAMPLE:

- Kathleen Feeney
- Head of Archives Processing and Digital Access
- kefeeney@uchicago.edu
- (no phone)
- Approves encoding practices

**Comment by MomoMoses** [comment]
For better admin UI, should the contact info be on a second row?

**Comment by johnjung** [comment]
Thanks very much, Laurie. Could you describe what you mean by "create a primary and secondary data column. Secondary column may be left blank." Does that mean that each Archive can have a primary and a secondary contact? And that there will always be at least one contact, but never more than two? Or is this something else? 

Also, what do you mean about putting the contact info in a second row?

**Comment by MomoMoses** [comment]
My thought was that there might be two contacts that would be equally good, but that more than that is a bit fussy and overkill. So right, most would have one contact, many could have two, but it shouldn't be required to have both filled in. Only one is required.

A second row was simply a thought about how to best display this information from the Wagtail page. Seems like a lot of info to fit in! Open to your ideas on this. :)

**Comment by johnjung** [comment]
That's great- thanks very much Laurie. Also, what does "approves encoding practices" mean? Is this a boolean, and will there be people who either do or do not approve encoding practices?

**Comment by MomoMoses** [comment]
No no, this is just human-readable information -- if there is someone who
has authority over how the finding aid is encoded, they should be a contact
and that role should be specified. It should not be any kind of required
element to check yes or no.

**Comment by johnjung** [comment]
Since this field can contain free-form information, would it be appropriate for me to call it "Notes?" (If that sound good to you, but there are specific types of things you'd like to add to some help text, please let me know how that help text should go.)

**Comment by MomoMoses** [comment]
Yes,  that's fine.  Help text could just say "indicate responsibility for
collections and role in the archives." And it would be optional,  could be
left blank
